tests/wrappers/test_add_inputs_wrapper.py

```diff
@@ -2,18 +2,19 @@
 """
 Tests for the AddInputsWorkChain.
 """
-
 # pylint: disable=unused-argument,redefined-outer-name
+import pytest
 
 from aiida import orm
 from aiida.engine import run_get_node
 
 from aiida_optimize.wrappers import AddInputsWorkChain
 
-from sample_processes import echo_process, EchoDictValue, EchoNestedValues  # pylint: disable=import-error,useless-suppression, unused-import
+from sample_processes import EchoDictValue, EchoNestedValues  # pylint: disable=import-error,useless-suppression, unused-import
 
 
-def test_basic(configure_with_daemon, echo_process):
+@pytest.mark.usefixtures('aiida_profile_clean')
```

Comment:
Can we `autouse` this fixture?

Reply:
I think only tests that involve actually running the calculation need to clean the profile after finished.